pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_getting_started/guide.fr-fr.md

```diff
@@ -0,0 +1,138 @@
+---
+title: "Guides premiers pas avec IAM dans mon cloud privée VMware managé par OVHcloud"
+excerpt: "Comment fonctionne IAM dans un cloud privé VMware managé par OVHcloud et une faq"
+updated: 2024-05-24
+---
+
+<style>
+.w-400 {
+  max-width:400px !important;
+}
+.h-600 {
+  max-height:600px !important;
+}
+</style>
+
+
+# Table des matières
+1. [Objectif](#Objectif)
+2. [Etape 1 - En pratique](#Enpratique)
+3. [Etape 2 - FAQ](#FAQ)
+4. [A suivre - Comment activer IAM](#Asuivre)
+5. [Aller plus loin](#Allerplusloin)
+
+---
+## Objectif
+**Dans ce guide, nous allons vous expliquer comment démarrer avec IAM dans votre Cloud Privée VMware sur OVHcloud et voir quelques questions réponses dans la FAQ** 
+
+---
+## Etape 1 : En pratique
+
+## Premiers pas avec IAM et mon cloud privée VMware managé par OVHcloud
+
+Pour utiliser IAM dans votre cloud privé, il va faut : 
+
+- [x] Activer IAM.
+
+- [x] Créer une politique IAM globale liée à mon identité OVHcloud.
+
+Des étapes supplémentaires sont nécessaires, nous allons les détailler dans la suite de ce guide.
+
+### Comment fonctionne IAM dans un cloud privée VMware sur OVHcloud ?
+> [!IMPORTANT]
+>
+> Un rôle IAM Vsphere ne peut (à ce jour) pas être ajouté grâce aux groupes de permissions managées.
+> Mais uniquement manuellement, voir le guide : [Comment ajouter un rôle IAM manuellement à une politique globale](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_role_policy)
+
+Gérer les identités de vos utilisateurs et de vos applications, ainsi que leurs droits d’accès dans une interface unique pour tous vos services. OVHcloud IAM permet de gérer finement l’ensemble des autorisations des produits OVHcloud et d’obtenir un meilleur niveau de sécurité avec une vision centralisée de vos accès.
+
+Pour simplifier, un rôle IAM remplace ainsi un utilisateur local Vsphere au sein de notre cloud privée. Et une politique permet d'associer votre identité OVHcloud à ce rôle.
+
+Voici les éléments nécessaires à une politique globale :
+- Produits : **Vsphere / VMware (hosted private cloud, service pack)**
+- Ressources : **PCC-XXX**
+- Actions : **Groupes de permissions, Ajoutées manuellement -> Rôle Vsphere/IAM**
+- Utilisateurs : **Utilisateur 1/2/3**
+
+<details>
+<summary>Schema IAM
+
+</summary>
+
+<summary> Voir le diagramme du fonctionnement de IAM avec un cloud privé OVHcloud :</summary> 
```

Comment:
check again the right `<details>` format on our internal pages.
This needs to be fixed and this format only used on rare occasions.
All the images links have also to be written in HTML if the `<details>` format is used.